### PR TITLE
WIP: Bugfix/multi group dropdown

### PR DESCRIPTION
**Pull request by giancorzo**
https://developertown.visualstudio.com/SIL%20International%20Scriptoria/_workitems/edit/28164

- [x] Create multi input dropdown
- [x] Add tests

### CODE REVIEW

File: source/SIL.AppBuilder.Portal.Frontend/src/ui/components/inputs/multi-group-select/styles.scss [hunk 1]
```diff
@@ -0,0 +1,13 @@
+@import '../../../styles/_base/variables.scss';
```
Comment (NullVoxPopuli) [inline source/SIL.AppBuilder.Portal.Frontend/src/ui/components/inputs/multi-group-select/styles.scss]: I wonder if we can setup sass aliases for this, like we can with js

File: source/SIL.AppBuilder.Portal.Frontend/src/ui/components/inputs/multi-group-select/with-data.tsx [hunk 2]
```diff
@@ -0,0 +1,156 @@
+import * as React from 'react';
+import { compose } from 'recompose';
+import { query, defaultOptions, GROUPS_TYPE, GROUP_MEMBERSHIPS_TYPE, USERS_TYPE, ORGANIZATIONS_TYPE, relationshipFor, attributesFor } from '@data';
```
Comment (NullVoxPopuli) [inline source/SIL.AppBuilder.Portal.Frontend/src/ui/components/inputs/multi-group-select/with-data.tsx]: this could be a multi-line dealio

**Comment by NullVoxPopuli** [comment]
@giancorzo apologies for the conflicts. :-\

**Comment by giancorzo** [comment]
I'm going to close this PR because it no longer applies.

**Pull request closed without merging**